charts/medusa-operator/templates/deployment.yaml

```diff
@@ -11,6 +11,10 @@ spec:
   template:
     metadata:
       labels: {{- include "k8ssandra-common.labels" . | indent 8 }}
+      {{- with .Values.podAnnotations }}
```

Comment:
Shouldn't `.podAnnotations` be under `annotations`?

Reply:
@jsanda Could you check again? I updated some commits based on your comments.